### DISCUSSION TITLE
Some Minecraft configuration settings not being applied to the server.

### DISCUSSION THREAD

**Discussion by spagheddie0** [itzg/docker-minecraft-server#2984] [Q&A]
I'm running Minecraft on TrueNAS. When I start the server, it has default settings regardless of which settings were chosen beforehand. 
If I select the gamemode as Creative when first creating the server, it still sets the gamemode as survival. When I look at the logs, it says "Default game type: SURVIVAL", despite the selected gamemode being creative.

**Comment by itzg** [comment]
Gamemode, difficulty, and few other properties only apply when the world is initially generated. You will need to use a server command to change those after world generation.
